test/e2e/chaos_test.go

```diff
@@ -309,6 +266,7 @@ func testSingleMemberRecovery(parentCtx context.Context, client crclient.Client,
 			Namespace:     manifests.HostedControlPlaneNamespace(cluster.Namespace, cluster.Name),
 			LabelSelector: labels.Set(etcdSts.Spec.Selector.MatchLabels).AsSelector(),
 		})
+		g.Expect(err).NotTo(HaveOccurred(), "failed to get etcd pods")
```

Comment:
I am strongly biased against Ginkgo/Gomega, but I will note that I have never seen so many missed error checks as in a codebase leaning on Gomega for assertions!